### DISCUSSION TITLE
Stylex with Jest

### DISCUSSION THREAD

**Discussion by prc5** [facebook/stylex#234] [Q&A] [answered]
Hey again!

I've tried to run test on the component using `defineVars`. I got this error:

```
stylex.defineVars should never be called. It should be compiled away.
```

It's not a surprise, there was no setup from my side on the jest.config.ts to support StyleX.

**How to provide setup for the StyleX?**
I know that there is `test` flag in the StyleX plugins. The problem is on how to connect it 😄 
My first idea was to provide some babel transform option with the stylex plugin or something like that 😅 
Do you have any idea or examples on how to make stylex work with the jest tests?

**Comment by nmn** [comment, marked as answer]
You just need a `.babelrc.js` file at the root of your project. Jest will pick it up.

Set `test: process.env.NODE_ENV === 'test'`
